src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -110,6 +110,27 @@ void print_pci_info(std::ostream &ostr)
     }
 }
 
+int xrt_xbutil_version_cmp() 
+{
+    /*check xbutil tools and xrt versions*/
+    std::string xrt = sensor_tree::get<std::string>( "runtime.build.version", "N/A" ) + "," 
+        + sensor_tree::get<std::string>( "runtime.build.hash", "N/A" );
+    if ( std::strcmp( xrt.c_str(), xcldev::driver_version("xocl").c_str() ) != 0 ) {
+        xocl::shim::xclLog(XRT_ERROR, "XRT", "Mixed versions of XRT and xbutil are not supported. \
+            \nPlease install matching versions of XRT and xbutil. \n");
```

Comment:
please also mention the environment variable to let developer know how to bypass the check.